flask_admin/static/vendor/bootstrap-daterangepicker/daterangepicker.js

```diff
@@ -58,7 +58,7 @@
                       '<input class="input-mini" type="text" name="daterangepicker_end" value="" />' +
                     '</div>' +
                     '<button class="applyBtn" disabled="disabled" type="button"></button>&nbsp;' +
-                    '<button class="cancelBtn" type="button"></button>' +
+                    '<button class="cancelBtn btn-danger" type="button"></button>' +
```

Comment:
Seems like it breaks normal displaying on bs3 mode, let's try to use `btn-secondary` instead of `btn-danger`, this class does not exist in bs3 and bs2, so it cant do any harm.